runtime/lua/vim/loader.lua

```diff
@@ -27,13 +27,17 @@ M.path = vim.fn.stdpath('cache') .. '/luac'
 ---@nodoc
 M.enabled = false
 
+---@nodoc
+M.charmap =
+  vim.tbl_add_reverse_lookup({ ['%'] = '%25', ['/'] = '%2F', [':'] = '%3A', ['\\'] = '%5C' })
```

Comment:
We want to remove `vim.tbl_add_reverse_lookup` since it's not easy to add type annotations for. Is it possible to remove this?